### PR TITLE
Log and case added

### PR DESCRIPTION
**Pull request by grammeaway**
Initial creation of log and case

### CODE REVIEW

File: src/domain/Case.java [hunk 1]
```diff
@@ -0,0 +1,112 @@
...
+
+import java.util.UUID;
+
+/**
+ *
+ * @author Victor Gram
+ */
+public class Case {
+
+  private UUID id;
```
Comment (kvartborg) [inline src/domain/Case.java]: Missing docblocks for attributes.

File: src/domain/Log.java [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+
+import java.util.Date;
+
+/**
+ *
+ * @author Victor Gram
+ */
+public class Log {
+
+  private Date date;
```
Comment (kvartborg) [inline src/domain/Log.java]: Missing docblocks for attribubtes

File: src/domain/Case.java [hunk 3]
```diff
@@ -0,0 +1,125 @@
+/**
+ * Case class, containing the data relevant for each individual case
+ */
+package domain;
+
+import java.util.UUID;
+
+/**
+ *
+ * @author Victor Gram
```
Comment (NJohansen) [inline src/domain/Case.java]: Remove author

File: src/domain/Log.java [hunk 4]
```diff
@@ -0,0 +1,148 @@
+/**
+ * Log class, creates log for each case
+ */
+package domain;
+
+import java.util.Date;
+
+/**
+ *
+ * @author Victor Gram
```
Comment (NJohansen) [inline src/domain/Log.java]: Remove author

**Pull request merged**